src/main/java/com/cjcrafter/armormechanics/ArmorMechanics.kt

```diff
@@ -201,6 +202,8 @@ class ArmorMechanics : JavaPlugin() {
     }
 
     companion object {
+        val DURABILITY_PRE = ChatColor.GRAY.toString() + "Durability: "
```

Comment:
All options must be configurable in the armor config, or in the central config.yml

Reply:
I'm making them configurable but for color support I've inspected me.deecaad.core.file.serializers.Serializer and saw plugin is using reflection to set colored lore (https://github.com/WeaponMechanics/MechanicsMain/blob/master/MechanicsCore/src/main/java/me/deecaad/core/file/serializers/ItemSerializer.java#L180-L190)
But I don't think I should do it using reflection for this because It means is will do reflection stuff on every armor damage event etc which is not good for performance. Is there any function that takes simply a String and returns a String that is usable for lores etc?

Reply:
Reflection isn't ideal, but it isn't as big of a deal as people make it out to be. I'd *argue* (not tested, just an argument) that getting `ItemMeta` is the far more expensive operaton in this case due to all the data being copied.

But honestly, you shouldn't  be making your own lore system. Check out the `me.deecaad.core.placeholder.PlaceholderHandler` class instead, which automatically handles placeholders.

Reply:
I understand the Placeholder system but does it have anything to do with lore? I had a look to default armors etc and didn't see anything related to placeholders.